src/input/ytdl_src.rs

```diff
@@ -14,7 +14,13 @@ use std::{
 use tokio::task;
 use tracing::trace;
 
+#[cfg(feature = "youtube-dl")]
+const YOUTUBE_DL_COMMAND: &str = "youtube-dl";
+#[cfg(feature = "youtube-dlc")]
+const YOUTUBE_DL_COMMAND: &str = "youtube-dlc";
+
 /// Creates a streamed audio source with `youtube-dl` and `ffmpeg`.
+#[cfg(any(feature = "youtube-dl", feature = "youtube-dlc"))]
```

Comment:
With one override feature, this check is unnecessary. This also stops `Restartable` sources from failing to compile.